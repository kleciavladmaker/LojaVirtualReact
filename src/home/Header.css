
header {
  display: flex;
  background-color: #4CAF50;
  color: #ffffff;
  padding: 15px;
}

header > h1 {
  flex-grow: 1;
  text-align: center;
}