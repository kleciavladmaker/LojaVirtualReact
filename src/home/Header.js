import React from 'react'
import logo from './logo.png'

import './Header.css'

export default function Header() {
  return (
    <header data-testid="header">
      <img src={logo} alt="ogo do site" />
      <h1>Loja Virtual</h1>
    </header>
  )
}