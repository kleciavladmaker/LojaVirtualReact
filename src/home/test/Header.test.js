import React from 'react'
import {render} from '@testing-library/react'

import Header from '../Header'

describe('<Header />', () => {
  test('Ao desenhar o componente, o titulo do site deve ser exibido', () => {
    const {getByText} = render(<Header />)
    const h1 = getByText('Loja Virtual')

    expect(h1).toBeInTheDocument()
  })

  test('Validar foto', () =>{
    const {getByTestId} = render(<Header />)
    const root = getByTestId('header')

    expect(root).toMatchSnapshot()
  })

  test.skip('1+1=2', ()=> {
    expect(1+1).toBe(2)
  }) 

  test.skip('2+2=4', () => {
    expect(2+1).toBe(4)
  }) 
})