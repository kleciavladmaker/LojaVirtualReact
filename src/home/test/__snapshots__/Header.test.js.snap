// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Validar foto 1`] = `
<header
  data-testid="header"
>
  <img
    alt="ogo do site"
    src="logo.png"
  />
  <h1>
    Loja Virtual
  </h1>
</header>
`;
